app/code/core/Mage/Wishlist/Block/Abstract.php

```diff
@@ -232,7 +232,7 @@ public function getEscapedDescription($item)
      */
     public function hasDescription($item)
     {
-        return trim($item->getDescription()) != '';
+        return ($item->getDescription() ?? '') != '';
```

Comment:
```suggestion
        return trim($item->getDescription() ?? '') != '';
```

I think the trim is still needed because otherwise it changes the behaviour of the function ;-)

Reply:
I have nothing against. If it is needed then a way must be found to skip that error message.

Reply:
it is needed because otherwise a description with only spaces will be marked as "having a description", which is a different behavior from the current one.

but the `?? ''` is totally correct